src/GovUk.Education.ExploreEducationStatistics.Content.ViewModels/PublicationSummaryViewModel.cs

```diff
@@ -12,6 +12,8 @@ public record PublicationSummaryViewModel
 
     public required string Slug { get; init; }
 
+    public string ReleaseSlug { get; init; } = string.Empty;
```

Comment:
Same comment as the one left in `PublicationSearchResultViewModel.cs` about possibly renaming this `LatestReleaseSlug`.